lib/src/analysis/incomplete_sib.rs

```diff
@@ -0,0 +1,68 @@
+use std::borrow::Cow;
+
+use telcom_parser::lte_rrc::{BCCH_DL_SCH_MessageType, BCCH_DL_SCH_MessageType_c1};
+
+use super::analyzer::{Analyzer, Event, EventType, Severity};
+use super::information_element::{InformationElement, LteInformationElement};
+
+pub struct IncompleteSibAnalyzer {
+    packet_num: usize,
+}
+
+impl Default for IncompleteSibAnalyzer {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
+impl IncompleteSibAnalyzer {
+    pub fn new() -> Self {
+        Self { packet_num: 0 }
```

Comment:
this seems common enough that we probably ought to just make the Harness handle it. i'll open an issue for this